.github/workflows/push-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Github Release
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      # wait for the build to succeed so that the manager image is available
+      - name: Wait for the 'push-telemetry-manager-build' job to succeed
```

Comment:
The name shouldn't be somethings like `Wait for the 'post-telemetry-manager-build-release' job to succeed` ?

Reply:
Good catch! yes, you are correct